doc/cephadm/install.rst

```diff
@@ -121,28 +121,41 @@ curl-based installation
 
      python3.8 ./cephadm <arguments...>
 
-* Although the standalone cephadm is sufficient to get a cluster started, it is
-  convenient to have the ``cephadm`` command installed on the host.  To install
-  the packages that provide the ``cephadm`` command, run the following
-  commands:
+.. _cephadm_update:
 
-  .. prompt:: bash #
-     :substitutions:
+update cephadm
```

Comment:
Update ?

Reply:
This PR is just a backport. The revisions will begin soon. And they will be extensive. And they will deal with exactly this kind of thing.